docs/guides/qiskit-code-assistant-vscode.mdx

```diff
@@ -0,0 +1,100 @@
+
+By default, the extension uses the `granite-8b-qiskit` model, which is listed in the Model Picker in the bottom status bar.
+
+![The bottom status bar is shown, with `granite-8b-qiskit` listed.](/images/guides/qiskit-code-assistant/vscode-selected-model.png)
+
+The first time you use the `granite-8b-qiskit` model a modal opens, listing some major restrictions that you should be aware of when using the model. Clicking `Accept` will accept the disclaimer and enable the model for code generation.
+
+![The window that opens upon first use contains restrictions tha must be accepted.](/images/guides/qiskit-code-assistant/eula.png)
+
+### Generate Code
+After accepting the EULA you can open a new file and start coding. To get a code suggestion, type `Ctrl+.` as shown:
+
+![A code suggestion is shown.  The suggested code is in grey.](/images/guides/qiskit-code-assistant/vscode-inline-suggestion.png)
+
+Press `Tab` to accept the model-generated suggestion or press `ESC` to reject it.
+
+<Admonition type="tip" title="Tips for working with code suggestions">
+- Type `Ctrl+.` again to generate additional suggestions.
+- When a suggestion is shown, you can use `Alt/Option+]` and `Alt/Option+[` to cycle forward and backward, respectively, through the suggestions (if available).
+- You can hover the cursor over the suggestion to use the pop-up toolbar to cycle through the suggestions, as shown:
+  ![The toolbar that pops up if you hover over suggested code is shown.](/images/guides/qiskit-code-assistant/vscode-inline-suggestion-toolbar.png)
+</Admonition>
+
+## Uninstall the VSCode extension
+
+To remove the Qiskit Code Assistant from VSCode, follow these steps:
+
+1. Launch Visual Studio Code.
+1. Click the Extensions icon.
+1. Search for `qiskit-vscode`.
+1. Click **Uninstall** for the "Qiskit Code Assistant".
+
+
+## Troubleshooting
+
+If you are not seeing the extension status bar in VSCode, check that the extension is installed and enabled under the extensions tab.
+
+If the extension is installed and enabled, but cannot select a model, verify that your current API Token has been added and the `Qiskit Code Assistant: Url` is properly set.
+
+## Contribute to the JupyterLab extension
+
+The code for this official extension is publicly available and open source. Check it out in [github.](https://github.com/Qiskit/qiskit-code-assistant-jupyterlab)
+
+
+## Next steps
+
+<Admonition type="tip" title="Recommendations">
+    - Check out examples to use the Qiskit Code Assistant for [circuits](https://docs.quantum.ibm.com/guides/circuit-library), [configuring error suppression](https://docs.quantum.ibm.com/guides/configure-error-suppression), and [transpiling with passmanagers.](transpile-with-pass-managers)
```

Comment:
This won't be love on Monday morning. Should we remove this section for now?

Reply:
Yep